.github/workflows/test.yml

```diff
@@ -25,7 +26,7 @@ jobs:
           - 20
 
     runs-on: ubuntu-latest
-    timeout-minutes: 10
+    timeout-minutes: 5
```

Comment:
this isnt specific to the restructuring, but noticed it while going through the details. 

ava is currently configured to timeout after 2 minutes, so 10 minutes just results in unnecessary waiting